portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,92 @@
+html{
+    background-color: rgb(17, 15, 15);
+    color: #fcf4f4;
+}
+
+body{
+    max-width: 800px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#navbar{
+    grid-area: header;
+    font-size: 20px;
+}
+
+#navbar .navbar-item{
+    padding:8px 16px;
+    float:left;
+    width:auto;
+    border:none;
+    display:block;
+    outline:0s;
+}
+
+.nav-button{
+    border:none;
+    display:inline-block;
+    padding:8px 16px;
+    vertical-align:middle;
+    overflow:hidden;
+    text-decoration:none;
+    color:inherit;
+    background-color:inherit;
+    text-align:center;
+    cursor:pointer;
+    white-space:nowrap
+}
+
+
+#nav-left-side{
+    float: left;
+}
+
+#nav-right-side{
+    float: right;
+}
+
+.clear{
+    clear: both;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin-top: 30%;
+    display: grid;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
styleguide suggests if you have multiple margin declarations to group them, so in this case it would be grouped into: 

margin: 30% auto 0px;

Look for "Margin - Shorthand Property" here: https://www.w3schools.com/css/css_margin.asp :)